_topic_maps/_topic_map.yml

```
@@ -993,6 +993,8 @@ Topics:
     File: cert-manager-operator-issuer-acme
   - Name: Configuring the egress proxy for the cert-manager Operator for Red Hat OpenShift
     File: cert-manager-operator-proxy
+  - Name: Authenticating the cert-manager Operator for Red Hat OpenShift with AWS
```

```suggestion
  - Name: Authenticating the cert-manager Operator for Red Hat OpenShift on AWS
```
PTAL at all similar references and make required change.